### ISSUE TITLE
Move contriution to another registration? (same participant)

### ISSUE THREAD

**Issue by magnolia61** [lcdservices/biz.lcdservices.attachcontribtoregistration#1]
Hi Brian, I'm impressed with all the handy extensions you made to make working with moving registrations and contributions easier. Would one of your extensions (maybe this one) also be able to move the contribution/payment from one registration to another (of the same participant)?

Sometimes someone registers and pays for event A but decided they rather go to event B.
I would think that this extension would come very close of being able to do so :-)

Cheers, Richard

**Comment by lcdservices** [comment]
No... it wouldn't support that currently. It explicitly looks for contributions that are not currently attached to an event and makes the action link available to them. But that would be a useful improvement. I'll look at what would be involved in doing that.

**Comment by lcdservices** [comment]
@magnolia61 --
I pushed some updates that will now support this. Please update with the latest code, test, and let me know if you see any issues.
